pkg/relayer/proxy/proxy.go

```diff
@@ -18,6 +19,11 @@ import (
 
 var _ RelayerProxy = (*relayerProxy)(nil)
 
+type (
+	serviceId       = string
+	RelayServersMap = map[serviceId][]RelayServer
```

Comment:
Does this need to be publically exposed?

Reply:
Not really, it's only used by the proxy package :+1: